Plugins/Flow.Launcher.Plugin.Program/Flow.Launcher.Plugin.Program.csproj

```diff
@@ -109,7 +109,6 @@
   </ItemGroup>
   
   <ItemGroup>
-    <PackageReference Include="Microsoft.Windows.SDK.Contracts" Version="10.0.18362.2005" />
```

Comment:
is this removal intended?

Reply:
Sorry, forget to response. Yes, because .Net 5 change the way of using RT. You can see on the TargetFramework of Program plugin is different from simply net5-windows but adding the version number. That replace the nuget package.